test/e2e/README.md

```diff
@@ -110,6 +110,32 @@ also need to copy the Antrea manifest to the master Docker container:
 go test -v github.com/vmware-tanzu/antrea/test/e2e -provider=kind
 ```
 
+## Running the performance test
+To run all benchmarks, without the standard e2e tests:
+```bash
+go test -v -timeout=30m -run=XXX -bench=. \
+    --performance.http.concurrency=16 \
+    github.com/vmware-tanzu/antrea/test/e2e 
+```
+The above command uses `-run=XXX` to deselect all `Test*` tests and uses `-bench=.` to select
+all `Benchmark*` tests. Since performance tests take a while to complete, you need to extend
+the timeout duration `-timeout` from the default `10m` to a longer one like `30m`.
+
+If you would like to run the performance tests in a different scale, you could run:
+```bash
+go test -v -timeout=30m -run=XXX -bench=BenchmarkCustomize \
+    --performance.http.requsts=5000 \
+    --performance.http.workloads=1000 \
+    --performance.http.concurrency=16 \
+    github.com/vmware-tanzu/antrea/test/e2e
+```
+
+All flags of performance tests includes:
+- `performance.http.concurrency (int)`: Number of allowed concurrent http requests (default 1)
+- `performance.http.requsts (int)`: Total Number of http requests
+- `performance.http.workloads (int)`: Number of CIDRs in the workload network policy
```

Comment:
I'm still unsure why we are sticking with the "workload" terminology, but the comment makes more sense now, thanks.

Reply:
Changed to `Number of CIDRs in the network policy`.